NEWS.md

```diff
@@ -1,3 +1,39 @@
+Printrun - 2.0.0
+====================
+
+Stable release supporting Python 3.10 and wxPython 4.2.
+
+See changes in previous pre-releases below for a full change log from previous
+stable release.
+
+### New Features
+
+ - Dropped support for Windows 32-bit executable (0c296ba)
+ - Added PrintrunGTK3 sub-module. Windows libraries (0c296ba)
+ - CairoSVG >= 2.6.0 not supported (0c296ba)
+ - Port to wxPython 4.2 (0c296ba)
+ - Pyglet >= 2.0 not supported (#1291)
+ - Added option for perspective transformation on gcode rendering (#1240)
+ - Port to Python 3.10 (#1224)
```

Comment:
I'd say: Various Python 3.10-related bugfixes (I can look them up in the Fedora specfile).

Reply:
Oh, I see, you list them individually later. That's probably fine.